lib/dialects/mssql/index.js

```diff
@@ -16,6 +16,7 @@ const ColumnCompiler = require('./schema/columncompiler');
 
 const SQL_INT4 = { MIN: -2147483648, MAX: 2147483647 };
 const SQL_BIGINT_SAFE = { MIN: -9007199254740991, MAX: 9007199254740991 };
+const MAX_UINT64 = 18446744073709551615;
```

Comment:
How did you get that numerical value? You know you cannot represent that big number exactly in javascript without using some special encoding like string or loosing precision.
```
> 18446744073709551615
18446744073709552000
> 
```

Reply:
Thats true... do you want me to include https://github.com/MikeMcl/bignumber.js for the check ?

Reply:
> > > https://github.com/tediousjs/tedious/blob/4120817754e91a32d9419093770771df590365ca/src/data-types/decimal.ts#L75
> > 
> > 
> > Cannot it be fixed in tedious if the root cause is there?
> 
> I don't think so... see [tediousjs/tedious#1058](https://github.com/tediousjs/tedious/issues/1058)

Actually now I investigated that tedious issue and few other related ones and it should be possible to fix in tedious (actually probably pretty easily by just writing value for example in 32bit pieces instead of trying to create one big value that might get too big...), but right now they just don't care to fix it since they want to write bigger reform how they are handling types. Anyways in this case if writing number always as `double` works it should be also more performant.

Reply:
Writing the numbers as double may work, but it would need a table column migration or am I wrong?

Reply:
I don't think it should need any migrations at least currently writing to float table with decimals seems to work, so I don't believe it should work just fine also other way around.

Reply:
Writing floats works (see tests).